services/src/handlers/wms.rs

```diff
@@ -147,9 +147,10 @@ async fn get_map<C: Context>(
     let thread_pool = ThreadPool::new(1); // TODO: use global thread pool
 
     let config_tiling_spec = get_config_element::<config::TilingSpecification>()?;
-    let execution_context = ExecutionContext {
+    let execution_context = MockExecutionContext {
```

Comment:
What is the reason to use a mock context in our WMS/WFS?

Reply:
because the real context will be the next step (second part of the dataset PR that I splitted)